src/constants/constants.js

```diff
@@ -0,0 +1,5 @@
+export const THEME_LIGHT = 'light';
+export const THEME_DARK = 'dark';
+
+export const CONTACT_US_PAGE_LINK = '/aboutus';
```

Comment:
Maybe it will be better to use just "about" or something like "about-us" in path